Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/Account/Login.cs

```diff
@@ -9,7 +9,7 @@ public class Login
         public string Email { get; set; }
 
         [Required]
-        [StringLength(14, ErrorMessage = "Password must be alphanumeric including at least 1 uppercase letter and a special character with 8 to 14 characters", MinimumLength = 8)]
+        [StringLength(14, ErrorMessage = "Invalid Password", MinimumLength = 8)]
```

Comment:
There shall not be any separate message for an invalid password. that user will figure out. we can not prompt them on password format. even length of the password. all these should happen at the time of registration or change password. So for this model, the password shall have only required validation.